File: src/main/java/com/curseclient/mixin/accessor/network/AccessorSPacketSoundEffect.java
package com.curseclient.mixin.accessor.network;

import net.minecraft.network.play.server.SPacketSoundEffect;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(SPacketSoundEffect.class)
public interface AccessorSPacketSoundEffect {
    @Accessor("soundPitch")
    void setPitch(float value);
}

File: src/main/kotlin/com/curseclient/mixin/accessor/network/AccessorSPacketSoundEffect.kt
package com.curseclient.mixin.accessor.network

import net.minecraft.network.play.server.SPacketSoundEffect
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(SPacketSoundEffect::class)
interface AccessorSPacketSoundEffect {
    @Accessor("soundPitch")
    fun setPitch(
        value : Float
    )
}